internal/dag/dag_test.go

```diff
@@ -2939,11 +2954,11 @@ func TestDAGIngressRouteCycle(t *testing.T) {
 	})
 
 	want := make(map[hostport]*VirtualHost)
-	want[hostport{host: "example.com", port: 80}] = &VirtualHost{
-		Port:   80,
-		host:   "example.com",
-		routes: routemap(&Route{path: "/finance", object: ir2}),
-	}
+	// want[hostport{host: "example.com", port: 80}] = &VirtualHost{
```

Comment:
please don't comment out code, just delete it.

Reply:
Whoops. Looks like a leftover debug artifact. Thanks!